source/web-contents.lisp

```diff
@@ -15,13 +15,14 @@
 (defmethod set-zoom-factor ((web-contents web-contents) factor)
   (message
    web-contents
-   (format nil "~a.setZoomFactor(\"~f\")" (remote-symbol web-contents) factor)))
+   (format nil "~a.setZoomFactor(~a)" (remote-symbol web-contents) factor)))
 
 (export-always 'get-zoom-factor)
 (defmethod get-zoom-factor ((web-contents web-contents))
-  (message
-   web-contents
-   (format nil "~a.getZoomFactor()" (remote-symbol web-contents))))
+  (parse-number:parse-number
+   (message
+    web-contents
+    (format nil "~a.getZoomFactor()" (remote-symbol web-contents)))))
```

Comment:
We could pull this dependency but isn't it overkill? We can use the reader since we won't need the extras that `parse-number` provides.

`(with-input-from-string (ratio "1.3") (read ratio)) => 1.3`

Reply:
Parse-number seems to be a much safer way to do this. I'd rather not use the reader for this!

Reply:
I think it's overkill but as you wish.